packages/suite/src/hooks/wallet/useSendFormImport.ts

```diff
@@ -60,38 +80,28 @@ export const useSendFormImport = ({ network, tokens, localCurrencyOption, fiatRa
                     }
 
                     // calculate Fiat from Amount
-                    if (fiatRate?.rate) {
-                        const cryptoValue = shouldSendInSats
-                            ? formatAmount(output.amount, network.decimals)
-                            : output.amount;
-                        output.fiat =
-                            toFiatCurrency(
-                                cryptoValue,
-                                output.currency.value,
-                                fiatRate,
-                                2,
-                                false,
-                            ) || '';
+                    if (rate) {
+                        const formattedAmount = new BigNumber(
+                            shouldSendInSats
+                                ? formatAmount(output.amount, network.decimals)
+                                : output.amount,
+                        );
+                        const fiatValueBigNumber = formattedAmount.multipliedBy(rate);
+
+                        output.fiat = fiatValueBigNumber.toFixed(2) || '';
                     }
-                } else if (
-                    Object.keys(fiatCurrencies).find(c => c === currency) &&
-                    fiatRate?.rate
-                ) {
+                } else if (Object.keys(fiatCurrencies).find(c => c === currency) && rate) {
```

Comment:
```suggestion
                } else if (Object.keys(fiatCurrencies).find(currency => currency === itemCurrency) && rate) {
```
or `outputCurrency`

you have to rename `currency` at the top

Reply:
a1718ae7de5e8b020f8b1c9b763eee8039cc6bd1